webpack.config.js

```diff
@@ -16,6 +16,11 @@ module.exports = {
   },
   devServer: {
```

Comment:
Make sure you need this please , 
i think `concurrently` package cover this configurations.

Reply:
With this code, the frontend project runs on 3000, the server runs on 8080.  This is cut from the log that prints out in the linux termin after running npm run dev:
[1] Node/Express server running on port 8080
[0] ℹ ｢wds｣: Project is running at http://localhost:3000/

Without this code, the frontend project is running on 8081:
[1] Node/Express server running on port 8080
[0] ℹ ｢wds｣: Project is running at http://localhost:8081/

I'm not sure if there is any downside to doing it either way, please let me know which one to keep.